### PR TITLE
[BUG FIX] add proper error message when PAT is incorrect.

### LINKED ISSUE #1281: CLI `setup` command give 401 error with incorrect PAT
**Describe the bug:**
```
? Enter organization name
 abrig
? Enter project name
 BedrockRocks
? Enter your AzDO personal access token
 ****************************************************
? Would you like to create a sample application repository? Yes
? Do you want to create a service principal? Yes
info:    attempting to login to az command line
info:    Successfully login to az command line
? Select one of the subscriptions
REDACTED
info:    attempting to create service principal with az command line
info:    Successfully created service principal with az command line
? Enter Azure Container Register Name. The registry name must be unique within Azure
 andrebriggs
azdo_org_name=abrig
azdo_project_name=BedrockRocks
azdo_pat=*********
az_create_app=true
az_create_sp=true
az_sp_id=REDACTED
az_sp_password=********
az_sp_tenant=REDACTED
az_subscription_id=REDACTED
az_acr_name=andrebriggs
az_storage_account_name=
az_storage_table=
workspace: quick-start-env
Project Created: no
High Level Definition Repo Scaffolded: no
Helm Repo Scaffolded: no
Sample App Repo Scaffolded: no
Manifest Repo Scaffolded: no
HLD to Manifest Pipeline Created: no
Service Principal Created: yes
Resource Group Created: no
Lifecycle Pipeline Created: no
Build Pipeline Created: no
ACR Created: no
Storage Account Created: no
Storage Table Created: no
Error: Error: Failed request: (401)
Status: Incomplete
error:   Error: Failed request: (401)
```

The issue is rooted in `webAPI.getCoreApi()` call in `setup.ts`.

**To Reproduce:**
Run `spk setup` with an invalid  or incorrect PAT

### PR DESCRIPTION
**Pull request by dennisseah**
closes: https://github.com/microsoft/bedrock/issues/1281

this is what we get when PAT is incorrect
```
code: 1000
message: setup-cmd-failed: Setup command was not successfully executed.
  code: 1400
  message: setup-cmd-core-api-err: Could not get Core API client. Check the Azure credential.
  details: Failed request: (401)
```

### CODE REVIEW

File: src/lib/i18n.json [hunk 1]
```diff
@@ -20,6 +20,9 @@
     "setup-cmd-prompt-err-no-subscriptions": "No subscriptions found.",
     "setup-cmd-prompt-err-subscription-missing": "Subscription Identifier was missing.",
     "setup-cmd-prompt-err-input-file-missing": "{0} did not exist or not accessible. Make sure that it is accessible.",
+    "setup-cmd-core-api-err": "Could not get Core API client. Check the Azure credential",
```
Comment (mtarng) [inline src/lib/i18n.json]: Core API is also for Azure DevOps. Let's be specific so we don't confuse the user between Azure and Azure DevOps here.

**Review by yradsmikham — APPROVED** [review]

File: src/lib/i18n.json (hunk 1 above)
Comment (dennisseah) [reply src/lib/i18n.json]: can you suggest a good message? thanks

Comment (mtarng) [reply src/lib/i18n.json]: The same message as the other errors:

Something like 
`Could not initialize Core API client. Check the Azure DevOps credentials.`

This is just being explicit about Azure vs Azure DevOps

Comment (dennisseah) [reply src/lib/i18n.json]: actually, this one is for Azure and not Azure DevOps. the other 2 are for Azure DevOps. We are using PAT to authenticate for this one and using Service Principal for the other 2.

**Review by andrebriggs — APPROVED** [review]

**Comment by dennisseah** [comment]
integration test results
```
fabrikam-hld-to-fabrikam-manifests has pipeline id of 423
Get the build status for build...
pipeline: fabrikam-hld-to-fabrikam-manifests-423:
pipeline_status this iteration --> "completed"
Successful build for pipeline: fabrikam-hld-to-fabrikam-manifests-423!
info:    Generating config.yaml from yaml file input.
info:    config.yaml was generated and located at /Users/veseah/.spk/config.yaml.
info:    Successfully initialized the spk tool!
Start Time           Service      Deployment   Commit  Src to ACR Image Tag                                             Result ACR to HLD Env    Hld Commit Result HLD to Manifest Result
4/6/2020, 8:17:28 PM fabrikam2019 51fb5e6d1b4f 52eebd5 711        fabrikam2019-fabrikam.acme.frontend:master-20200407.1 ✓      711        MASTER 4d88c1f    ✓      712             ✓     
1 deployment(s) were returned by spk deployment get
Verified that srcPipeline:711 acrPipelineId:711 hldPipelineId:712 pullrequest:"368" are all linked together correctly.
Successfully reached the end of spk deployment get tests.
```

**Pull request merged**